src/renderer/ui/audio.js

```diff
@@ -5,15 +5,16 @@ let audio_sfx_volume = 100
 let audio_msx_volume = 100
 let muted            = false
 
-function render(){
+function render(game){ 
```

Comment:
game can be imported so its not necessary to pass along

Reply:
ill try that, should i change audio to typescript either?

Reply:
Convert it all over to Typescript